app/src/pages/outside/loginPage/pageBlocks/serviceVersionsBlock/serviceVersionBlockTooltip/serviceVersionsBlock.jsx

```diff
@@ -17,58 +17,16 @@
 import classNames from 'classnames/bind';
 import PropTypes from 'prop-types';
 import { withTooltip } from 'components/main/tooltips/tooltip';
-import { FormattedMessage } from 'react-intl';
 import { useDispatch } from 'react-redux';
 import { hideModalAction, showModalAction } from 'controllers/modal';
 import styles from './serviceVersionsBlock.scss';
-import { ServiceVersionItem } from '../serviceVersionItem';
-import { ServiceVersionsBlockModal } from '../serviceVersionBlockModal';
+import { ServiceVersionsBlockModal } from '../serviceVersionBlockModal/serviceVersionBlockModal';
```

Comment:
```suggestion
import { ServiceVersionsBlockModal } from '../serviceVersionBlockModal';
```
I suggest use shortened imports when it's possible to simplify reading and avoid a lot of renaming in case of changing the inner files/folders structure.
Please check all places.

Reply:
Done.